packages/client/modules/userDashboard/containers/OrgMembers/OrgMembersRoot.tsx

```diff
@@ -20,16 +21,26 @@ interface Props {
 
 const OrgMembersRoot = (props: Props) => {
   const {orgId} = props
-  const atmosphere = useAtmosphere()
+  const queryRef = useQueryLoaderNow<OrgMembersRootQuery>(orgMembersRootQuery, {
+    orgId,
+    first: 10000
+  })
   return (
-    <QueryRenderer
-      environment={atmosphere}
-      query={query}
-      variables={{orgId, first: 10000}}
-      fetchPolicy={'store-or-network' as any}
-      render={renderQuery(OrgMembers, {size: LoaderSize.PANEL})}
-    />
+    <Suspense fallback={''}>{queryRef && <OrgMembersContainer queryRef={queryRef} />}</Suspense>
```

Comment:
for these Suspense `fallback` here, not quite sure if I need to add back the `Loader.Size` Loader component at this level

Reply:
-1 I think we should have a `MockList` fallback here